src/libcollections/btree/map.rs

```diff
@@ -962,14 +1015,15 @@ impl<K, V> IntoIterator for BTreeMap<K, V> {
         IntoIter {
             front: first_leaf_edge(root1),
             back: last_leaf_edge(root2),
-            length: len
+            length: len,
         }
     }
 }
 
 impl<K, V> Drop for IntoIter<K, V> {
     fn drop(&mut self) {
-        for _ in &mut *self { }
+        for _ in &mut *self {
```

Comment:
This should be a bug on rustfmt (I don't really find it ugly, but meh), not a comment here.